generator/README.md

```diff
@@ -0,0 +1,95 @@
+# Custom Resources Generator
+
+Up until release v0.12 metallb was configured via metallb's Configmap.
+Deprecating the configmap impacts on the users' current configurations.
+To mitigate this, the MetalLB resource generator is an offline tool to
+convert the configmap into the new CRDs.
+
+## Using the generator
+
+### Prerequisites
+
+To use the generator, first create the container image:
+```
+inv build-cr-generator
+```
+### Procedure
+
+Run the container and redirect the output to a file:
+```
+docker run -v <PATH_TO_CONFIG>:/var/input <IMAGE_ID> -origin <CONFIGMAP_FILE_NAME> > metallb_resources.yaml
+```
+
```

Comment:
I think we would build this image and push it ourselves, the user won't have to build.
IMAGE_ID will be something like metallb/generator